### ISSUE TITLE
Does this plugin still work on nodebb V2.8.6

### ISSUE THREAD

**Issue by bhtools** [julianlam/nodebb-plugin-solr#77]
I have install Solr
and the plugin:  nodebb-plugin-solr

 i active the "nodebb-plugin-solr" ,and rebuild and restart my nodebb

the i found nothing in plugins areas.

i‘m not sure does this plugin still works

thanks!

**Comment by bhtools** [comment]
maybe there is some additional imformations:
i install  nodebb-plugin-solr from plugin web.  i search it .
i realized that the version is  3.0.1。

**Comment by julianlam** [comment]
Is your forum in Chinese? I am guessing because you used this symbol: 。

If so solr will not work for you as Chinese (and other Asian characters) are not indexed properly. You will want to try using [Meilisearch](https://github.com/oplik0/nodebb-plugin-meilisearch)

**Comment by julianlam** [comment]
The plugin is quite outdated but it should still work for v2.x. You'll want to check for server-side errors to see what is happening when you try to start NodeBB.

**Comment by bhtools** [comment]
> 

yes ,  i try to install this plugin for chinese searching.  but it doesn't work.

but i found ,the nodebb plugin web shows the latest version is 3.0.1,but on github,it is 5.3.1.

i try to use "npm install nodebb-plugin-solr" and then the version is correct.  5.3.1.

but when i  activate this plugin，my admin dashboard will   502 error，i  don't know why?

thanks for your reply!
